go/vt/vttablet/onlineddl/executor.go

```diff
@@ -3799,17 +3799,21 @@ func (e *Executor) deleteVReplicationEntry(ctx context.Context, uuid string) err
 }
 
 // gcArtifactTable garbage-collects a single table
-func (e *Executor) gcArtifactTable(ctx context.Context, artifactTable, uuid string, t time.Time) (string, error) {
+func (e *Executor) gcArtifactTable(ctx context.Context, artifactTable, uuid string, timestampInThePast time.Time) (string, error) {
```

Comment:
nit: it seems weird to rename this function parameter. Those are the semantics the caller is using, this function simply generates the rename statement with whatever time is given to it.

Reply:
renamed back to original generic name `t`